hack/pipeline.dockerfile

```diff
@@ -11,7 +11,7 @@ RUN cd kustomize && \
     go install .
 RUN ~/go/bin/kustomize version
 
-FROM quay.io/operator-framework/operator-sdk:v1.2.0
+FROM quay.io/operator-framework/operator-sdk:v1.3.0
```

Comment:
this didn't affect the change, but as the RMO osdk was to this version, I thought.. why not